pkg/resource/template/resource.go

```diff
@@ -257,7 +257,11 @@ func (t *TemplateResource) sync() error {
 				return err
 			}
 		}
-		log.Info("Target config " + t.Dest + " has been updated")
+		msg := "Target config " + t.Dest + " has been updated"
```

Comment:
Basic idea here is that we receive a lot of updates that we don't care about and just ignore in confd.

This is the point of time in the code that we know the file has been changed due to an update we just received. This PR passes through the key that triggered this update so we can log it out, which is useful when trying to identify what event triggered a change, but without logging out all of fluff events that don't cause any real work.